core/tx_pool.go

```diff
@@ -509,7 +509,7 @@ func (pool *TxPool) local() map[common.Address]types.Transactions {
 func (pool *TxPool) validateTx(tx *types.Transaction, local bool) error {
 	// Heuristic limit, reject transactions over 32KB to prevent DOS attacks
 	if tx.Size() > 32*1024 {
-		return ErrOversizedData
+		return fmt.Errorf("%v your txSize:%d;limitSize:%d", ErrOversizedData, tx.Size(), 32*1024)
```

Comment:
Some comment 🙂

1. The delimiter between _"ErrOversizedData"_ and _"your txSize"_ may be replaced to another delimiter(e.g. ":") instead of blank space.

2. _txSize:%d_ should be modified to _txSize:%f_

3. _"32*1024"_ appears twice here. Suggest to define it as a variable or constant.